modules/kafka/src/main/scala/higherkindness/mu/kafka/ConsumerStream.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017-2020 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package higherkindness.mu.kafka
+
+import cats.effect.{ConcurrentEffect, ContextShift, Sync, Timer}
+import fs2.Stream
+import fs2.kafka.{ConsumerSettings, KafkaConsumer}
+import higherkindness.mu.format.Decoder
+import io.chrisdavenport.log4cats.Logger
+import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
+
+object ConsumerStream {
+
+  def apply[F[_]: Sync, A](topic: String, settings: ConsumerSettings[F, String, Array[Byte]])(
+      implicit contextShift: ContextShift[F],
+      concurrentEffect: ConcurrentEffect[F],
+      timer: Timer[F],
+      decoder: Decoder[A],
+      sync: Sync[F]
```

Comment:
Not needed at this point

Reply:
The `: Sync` context bound is also not needed